docs/user/whois.rst

```diff
@@ -45,6 +45,17 @@ met:
 - There is **no existing WHOIS record** for that IP.
 - WHOIS lookup is **enabled** for the device's organization.
 
+Handling Existing Devices
```

Comment:
Instead of adding this section here, add it at the bottom of the setup instructions and provide a copy/paste example that includes enabling the virtualenv (for users using ansible-openwisp2).

Point 6 should be to restart the application if using ansible-openwisp2, while docker containers should just be respawned.

Point 7 should be to use the management command.